airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -62,13 +67,15 @@ def __init__(
         self,
         connections_prefix: str = 'airflow/connections',
         variables_prefix: str = 'airflow/variables',
+        configurations_prefix: str = 'airflow/configuration',
```

Comment:
```suggestion
        config_prefix: str = 'airflow/config',
```

Reply:
etc (haven't made this change everywhere)

Reply:
:D I went back and forth with this

Reply:
fixed